Spec/Auth.swift

```diff
@@ -382,7 +382,7 @@ class Auth : QuickSpec {
                 //  - authCallback and authUrl are both specified
                 let cases: [String: (ARTAuthOptions) -> ()] = [
                     "useTokenAuth and no key":{ $0.useTokenAuth = true },
-                    "authCallback and authUrl":{ $0.authCallback = { params, callback in /*nothing*/ }; $0.authUrl = NSURL(string: "http://auth.ably.io") as URL? }
+                    "authCallback and authUrl":{ $0.authCallback = { params, callback in /*nothing*/ }; $0.authUrl = URL(string: "http://auth.ably.io") as URL? }
```

Comment:
You can remove the `as URL?` too:

```
URL(string: "http://auth.ably.io") as URL?
                                   ^~~~~~~
```

Reply:
@ricardopereira good point :) https://github.com/ably/ably-ios/pull/797/commits/ae08377cef3deb1c1775f4cb002dcd8910146b11